db/migrate/20190520222453_create_users.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
```

Comment:
So, I believe that this says...
create a class of createUsers
define a function called change that will create a users table, include in that table a column called name, and add timestamps

Reply:
That's basically it. These migrations files are generated by Rails, but they are customizable.